faker/providers/person/vi_VN/__init__.py

```diff
@@ -0,0 +1,125 @@
+from .. import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    formats_female = (
```

Comment:
We need data source for names. Please refer to our [coding style](https://github.com/joke2k/faker/blob/master/docs/coding_style.rst#data-sets)